app/assets/status/approved-copy.svg

```diff
@@ -0,0 +1,215 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
All of the SVGs should be squashed with `svgo` or something similar.  You can run `make compress-images`.